week-2/implement/vat.js

```diff
@@ -8,3 +8,11 @@
 // Given a number,
 // When I call this function with a number
 // Then it returns the new price with VAT added on
+
+function calculatePriceWithVAT(price) {
+  price = price * 1.2;
```

Comment:
Works, Could just do return price * 1.2; here - this line isn't strictly necessary